draft-ietf-webtrans-http3.md

```diff
@@ -218,6 +218,27 @@ session have alternative mechanisms:
   limiting {{!RFC6585}}.  Unlike the previous method, this signal is directly
   propagated to the application.
 
+## Prioritization
+
+WebTransport does not provide any priority signalling mechanism for streams and
+datagrams within a WebTransport session; such mechanisms will be defined by
+application protocols using WebTransport.
+
+{{!RFC9218}} defines the Priority header which MAY be set in a WebTransport
+CONNECT request or response.  This header SHOULD govern the scheduling of all
+data within the WebTransport session, including WebTransport streams and
+datagrams.
```

Comment:
It seems like you're making new rules for RFC 9218 here. You don't mention the frame (which can only go in one direction) and you're too ambiguous about how these signals affect the dataflows. When a client sends the signal, what is the prioritization affecting (local streams or remote streams?) and vice versa. 

I'm also not sure how this is supposed to gel with the W3C WebTransport API that doesn't allow control over these fields.

Reply:
If I'm understanding correctly, this is saying that you must prioritize everything associated with a given WebTransport session at the same level. Technically, that's not required since we broke out each stream into a "native" H3 stream, but that seems like more of a bug than a feature, since interleaving priorities between streams of different WebTransport sessions seems like a recipe for pain. Instead, we probably want to say that everything must stay within the session, that session has a fixed priority, and if you want to prioritize within the session, it's up to you to send the right data in the right order (see the W3C API for that).